CHARTER.adoc

```diff
@@ -66,7 +77,7 @@ The TODO Group shall be conducted so as to:
  .. Under no circumstances shall The Linux Foundation be expected or required to undertake any action on behalf of TODO that is inconsistent with the tax exempt purpose of The Linux Foundation.
  . *Intellectual Property Policy*
  .. All contributions shall be accompanied by a Developer Certificate of Origin sign-off (http://developercertificate.org).
- .. All new inbound code contributions to TODO shall be made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0) or an alternative OSI-approved open source license clearly identified by the contributor in the contribution in accordance with the DCO. All documentation will be contributed to and made available to TODO under the Creative Commons Attribution 4.0 International License (available athttp://creativecommons.org/licenses/by/4.0/) or an alternative OSI-approved open source license clearly identified by the contributor in the contribution in accordance with the DCO.
+ .. All new inbound code contributions to TODO shall be made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0) or an alternative OSI-approved open source license clearly identified by the contributor in the contribution in accordance with the DCO. All documentation will be contributed to and made available to TODO under the Creative Commons Attribution 4.0 International License (available at http://creativecommons.org/licenses/by/4.0/) or an alternative OSI-approved open source license clearly identified by the contributor in the contribution in accordance with the DCO.
  .. All outbound code will be made available under the license for the inbound contribution.
  .. Any TODO member may raise an objection or concern regarding any contribution made under an alternative license to the Steering Committee for resolution.
```

Comment:
```suggestion
 .. Any TODO General Member may raise an objection or concern regarding any contribution made under an alternative license to the Steering Committee for resolution.
```

Reply:
What is a contribution as far as a DCO is concerned? Should we specify code here? I have never executed a DCO and I am clearly contributing to TODO by adding to this repo.